distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/carbon.xml.j2

```diff
@@ -633,40 +633,12 @@
            The total file upload size limit in MB
         -->
         <TotalFileSizeLimit>{{server.file_upload.file_size_limit}}</TotalFileSizeLimit>
-
-        <Mapping>
```

Comment:
Delete these sections from the carbon.xml as well

Reply:
Since EI team has some use cases with DBS file uploader, we did not remove DBS file uploader class. Since DBS class is mapped here, don't we need the file size limit tag as well?

Reply:
Since EI team also not going to use other uploaders than  DBS file uploader, we can remove them from carbon.xml as well. 

We need the  file size limit tag, I was suggesting to remove only the Mapping tags from carbon.xml

Reply:
updated the corbon.xml files